src/algebra/gcd_monoid/integrally_closed.lean

```diff
@@ -0,0 +1,57 @@
+/-
+Copyright (c) 2022 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import algebra.gcd_monoid.basic
+import ring_theory.integrally_closed
+import ring_theory.polynomial.scale_roots
+import ring_theory.polynomial.eisenstein
+
+/-!
+
+# GCD domains are integrally closed
+
+-/
+
+open_locale big_operators polynomial
+
+lemma is_localization.surj_of_gcd_domain {R : Type*} [comm_ring R] [is_domain R] [gcd_monoid R]
```

Comment:
You should change the name, since there is no `surjective` in the statement.

Reply:
I think the name shouldn't be that confusing since it reflects `is_localization.surj'`, whose type signature is basically the same but without the `gcd` constraint?